--- d:/dev/winmerge/src/SelectPluginDlg.cpp	Wed Apr 27 00:31:29 2022
+++ c:/dev/winmerge/src/SelectPluginDlg.cpp	Fri Apr 29 17:06:36 2022
@@ -113,6 +113,12 @@
 {
 	CTrDialog::OnInitDialog();
 
+	// setup handler for resizing this dialog	
+	m_constraint.InitializeCurrentSize(this);
+	m_constraint.SubclassWnd(); // install subclassing
+	// persist size via registry
+	m_constraint.LoadPosition(_T("ResizeableDialogs"), _T("SelectPluginDlg"), false);
+
 	m_bNoExtensionCheck = GetOptionsMgr()->GetBool(OPT_PLUGINS_UNPACK_DONT_CHECK_EXTENSION);
 	m_bOpenInSameFrameType = GetOptionsMgr()->GetBool(OPT_PLUGINS_OPEN_IN_SAME_FRAME_TYPE);
 

